### PR TITLE
Add minimal citation file

### LINKED ISSUE #350: Add a citation file
I just saw an annoucement on Twitter that GitHub now provides a feature to facilitate citations of software directly from the repo page:

https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-citation-files

We'd need to create an appropriate [citation file](https://citation-file-format.github.io/) and GitHub will add a "Cite this repository" section on the right handside of the repo page.

### PR DESCRIPTION
**Pull request by ghisvail**
About citation files integration on GitHub: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files

Check the `Cite this repository` section on the right handside of the GitHub interface for this branch.

Closes #350

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,55 @@
+cff-version: 1.2.0
+message: If you use this software, please cite it as below.
+authors:
+  - family-names: Routier
+    given-names:  Alexandre
+    affiliation:
+  - family-names: Brugos
```
Comment (14thibea) [inline CITATION.cff:7]: Burgos

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    given-names: Ninon
+  - family-names: Diaz
+    given-names: Mauricio
+  - family-names: Bacci
+    given-names: Michael
+  - family-names: Bottani
+    given-names: Simona
+  - family-names: El-Rifai
+    given-names: Omar
+  - family-names: Fontella
```
Comment (14thibea) [inline CITATION.cff:17]: Fontanella

File: CITATION.cff [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    given-names: Tristan
+  - family-names: Samper-Gonzalez
+    given-names: Jorge
+  - family-names: Teichmann
+    given-names: Marc
+  - family-names: Thibeau-Sutre
+    given-names: Elina
+  - family-names: Vaillant
+    given-names: Ghislain
+  - family-names: WEN
```
Comment (14thibea) [inline CITATION.cff:43]: Wen

File: CITATION.cff [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    given-names: Junhao
+  - family-names: Wild
+    given-names: Adam
+  - family-names: Habert
+    given-names: Marie-Odille
```
Comment (14thibea) [inline CITATION.cff:48]: Marie-Odile

**Review by 14thibea — COMMENTED** [review]
I found some typos in names (I always check because my name is very often inaccurately written ^^)

**Comment by ghisvail** [comment]
> I found some typos in names (I always check because my name is very often inaccurately written ^^)

😱 Thanks!

**Comment by mdiazmel** [comment]
If no one else has remarks about how to spell its name :), I propose to merge this one quickly!

**Pull request merged**